src/website/src/pages/ec/components/timeline/docs/usage.mdx

```diff
@@ -32,17 +32,17 @@ import { Paragraph, Anatomy, Link } from '@ecl/website-components';
 
 ### Default
 
-| Elements  | Mandatory | Description                                                       |
-| --------- | --------- | ----------------------------------------------------------------- |
-| heading   | yes       | heading of the item                                               |
-| timestamp | yes       | timestamp when the item happened (or is scheduled to)             |
-| details   | no        | a more detailed description. Can include text, links, media files |
+| Elements  | Mandatory Desktop | Mandatory Mobile | Description                                                       |
+| --------- | ----------------- | ---------------- | ----------------------------------------------------------------- |
+| heading   | yes               | yes              | heading of the item                                               |
+| timestamp | yes               | yes              | timestamp when the item happened (or is scheduled to)             |
+| details   | no                | no               | a more detailed description. Can include text, links, media files |
 
-### Conditional
+### Conditional (over 12 items)
 
-| elements      | Mandatory | Description                                                                                                                                                          |
-| ------------- | --------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| expand button | yes       | actionable button to display the full list of items in the timeline - to be used **if** there are over 12 items in the timeline<br/>**Position**: under the 3rd item |
+| elements      | Mandatory Desktop | Mandatory Mobile | Description                                                                                                         |
+| ------------- | ----------------- | ---------------- | ------------------------------------------------------------------------------------------------------------------- |
+| expand button | yes               | yes              | if there are over 12 items in the timeline then Actionable button to display the full list of items in the timeline |
```

Comment:
I understand we received this in the "specs" for this update, but is this sentence meaning anything to you?
_if there are over 12 items in the timeline then Actionable button to display the full list of items in the timeline_
Anyway...if that's what they want.

Ah, Paulo, this is one of those prs for which we request approval by the "business". Once we get it the way we want we won't merge, we will ask first for an approval by "who created the issue" ;)

Reply:
I agree, it's not their best sentence...